sdks/python/apache_beam/runners/interactive/interactive_beam_test.py

```diff
@@ -67,6 +77,31 @@ def test_watch_class_instance(self):
     test_env.watch(self)
     self.assertEqual(ie.current_env().watching(), test_env.watching())
 
+  def test_show_always_watch_given_pcolls(self):
+    p = beam.Pipeline(ir.InteractiveRunner())
+    # pylint: disable=range-builtin-not-iterating
+    pcoll = p | 'Create' >> beam.Create(range(10))
+    # The pcoll is not watched since watch(locals()) is not explicitly called.
+    self.assertFalse(
+        pcoll in _get_watched_pcollections_with_variable_names())
+    # The call of show watches pcoll.
+    ib.show(pcoll)
+    self.assertTrue(
+        pcoll in _get_watched_pcollections_with_variable_names())
+    # The name of pcoll is made up by show.
+    self.assertEqual(
+        'PCollection_Create/Map_decode_.None_',
```

Comment:
This test will break if anything changes related to this arbitrary naming. Maybe just check that it exists.

Reply:
Removed the str assertion. Kept the existence check.